contracts/src/Verifier.sol

```diff
@@ -9,34 +9,207 @@ import "./IVerifier.sol";
  * @title Verifier contract
  * @author Seiya Kobayashi
  */
-// TODO: enable solhint
-/* solhint-disable */
 contract Verifier is IVerifier, Ownable {
-    // TODO: add necessary types
+    // TODO: add types
 
-    // TODO: add necessary state variables
+    /*
+        state variables
+    */
 
-    // TODO: add necessary events
+    // array of content IDs of models
+    Hash[] private models;
+    // mapping of owner address to array of content IDs of models
+    mapping(address => Hash[]) private ownerAddressToModels;
```

Comment:
FYI: You can now use named parameters for mapping with solidity v0.8.18
https://blog.soliditylang.org/2023/02/01/solidity-0.8.18-release-announcement/
https://twitter.com/PaulRBerg/status/1621949932608360453?s=20&t=7v8y-BhaJhBEW1q3kU1n_Q